### PR TITLE
Margin pics

### PR DESCRIPTION
**Pull request by witgaw**
Add a script setting up market with conditions needed for margin screenshots included in the documentation.

The whole `visualisations` subfolder is just a suggestion, happy to move it elsewhere. Can also get rid of the readme if it's not useful.

Includes Tom's commit with the fix to the issue I was facing locally yesterday.

### CODE REVIEW

File: examples/visualisations/margins.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+wallets = [MM_WALLET, MM_WALLET2, TRADER_WALLET]
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
+
+    with VegaServiceNull(
+        run_with_console=True,
+        start_order_feed=True,
+        retain_log_files=True,
```
Comment (davidsiska-vega) [inline examples/visualisations/margins.py]: If things are working I'd set this to False; it's not just the log files but also vega home so chain, data node... and even the log can be GB - though probably not from this script.

**Review by davidsiska-vega — COMMENTED** [review]
I've left one comment but I'd wait for @TomMcL to review since it has a change in the `null_sevice.py` ...

File: examples/visualisations/margins.py (hunk 1 above)
Comment (witgaw) [reply examples/visualisations/margins.py]: Good call, thanks - got rid of these now.

**Review by TomMcL — DISMISSED** [review]
LGTM

**Review by TomMcL — APPROVED** [review]

**Pull request merged**